### ISSUE TITLE
Load Javascript resources in parallel (potential 2x cold page load time improvement)

### ISSUE THREAD

**Issue by djahandarie** [ether/etherpad-lite#5420]
![etherpad-resource-loading](https://user-images.githubusercontent.com/21412/154594847-25124020-d6c4-4ae0-82ff-200ca1e96475.png)

Currently, require-kernel.js loads resources sequentially. I have a <30ms roundtrip to the server this is on, and despite that it still takes a full 1.5s (out of the total 3s) to dynamically load all the javascript. I don't want to think about someone who has a high latency connection (I could imagine 10s+ load times just from this).

Could these resources be fetched in parallel?

**Comment by rhansen** [comment]
We're aware of the serial loading issue. Unfortunately, it's not easy to fix, and our time is limited. If this is important to you, you are welcome to send a PR yourself, or you can sponsor development.

In the meantime you might see some performance gains by using [ep_webpack](https://www.npmjs.com/package/ep_webpack).

**Comment by rhansen** [comment]
Duplicate of #4820
